common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/IIpcStrategy.java

```diff
@@ -16,7 +16,8 @@ enum Type {
         BOUND_SERVICE("bound_service"),
         ACCOUNT_MANAGER_ADD_ACCOUNT("account_manager_add_account"),
         CONTENT_PROVIDER("content_provider"),
-        LEGACY_ACCOUNT_MANAGER_FOR_BROKER_API("legacy_account_manager_for_broker_api");
+        LEGACY_ACCOUNT_MANAGER_FOR_BROKER_API("legacy_account_manager_for_broker_api"),
+        LEGACY_ACCOUNT_AUTHENTICATOR_FOR_WPJ_API("legacy_account_authenticator_for_wpj_api");
```

Comment:
What is the difference between account manager and account authenticator?

Reply:
legacy_account_manager_for_broker_api is for the LegacyAccountManagerForBrokerApiStrategy class
"Old mechanism to communicate to Broker for operation like: API hello, get broker accounts, and remove accounts"
legacy_account_authenticator_for_wpj_api will be used as a backup strategy for WPJ operations in case the main strategy (Broker content provider) is not available